ui/main/src/app/services/selectedCard/SelectedCardService.ts

```
@@ -0,0 +1,90 @@
+    public static getSelectedCard(): Observable<SelectedCard> {
+        return SelectedCardService.selectedCardWithChildrenChange.asObservable();
+    }
+
+    public static setCardDeleted(cardId: string): void {
+        if (SelectedCardService.selectedCardId === cardId) SelectedCardService.selectedCardDeleted.next(cardId);
+    }
+
+    public static getSelectedCardsDeleted(): Observable<any> {
+        return SelectedCardService.selectedCardDeleted.asObservable();
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)